android/src/com/mapswithme/maps/editor/EditorFragment.java

```diff
@@ -222,7 +222,6 @@ boolean setEdits()
     Editor.nativeSetHouseNumber(mHouseNumber.getText().toString());
     Editor.nativeSetZipCode(mZipcode.getText().toString());
     Editor.nativeSetBuildingLevels(mBuildingLevels.getText().toString());
-    Editor.nativeSetPhone(mPhone.getText().toString());
```

Comment:
Why is it removed? Are phones properly stored and sent to OSM?

Reply:
I moved `Editor.nativeSetPhone(phone);` call to `EditorHostFragment` class simmilar to cuisine, opening house, street editors. Becase I created `PhoneFragment` to edit multiple phones, this fragment has "Save" button